<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListeContacts</class>
 <widget class="QWidget" name="ListeContacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listeContacts"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAfficher">
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonEditer">
     <property name="text">
      <string>Editer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSupprimer">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
